src/pages/SubmissionHistory/SubmissionHistory.js

```diff
@@ -26,8 +28,9 @@ export const SubmissionHistory = () => {
   // helper for auto-refresh
   const refresh = Utils.withBusyState(setLoading, async () => {
     try {
-      const loadedRunSetData = await loadRunSetData(signal)
+      const loadedRunSetData = await loadAllRunSets(signal)
       setRunSetData(loadedRunSetData)
+      setRunSetsFullyUpdated(true)
```

Comment:
Should this be set based on the `fully_updated` value returned in run set similar to one in Submission Details page?

Reply:
Oops, it definitely should, good spot!